socorro/external/postgresql/graphics_devices.py

```diff
@@ -15,28 +15,32 @@ class GraphicsDevices(PostgreSQLBase):
 
     def get(self, **kwargs):
         filters = [
-            ("vendor_hex", None, "str"),
-            ("adapter_hex", None, "str"),
+            ("vendor_hex", None, ["list", "str"]),
+            ("adapter_hex", None, ["list", "str"]),
         ]
         params = external_common.parse_arguments(filters, kwargs)
         for key in ('vendor_hex', 'adapter_hex'):
-            if not params[key]:
+            param = params[key]
+            if not param or (len(param) == 1 and not param[0]):
```

Comment:
This is a side effect of ``parse_arguments``, which will generate ``['']`` when the argument was empty. It should probably actually be solved in the ``parse_arguments`` function instead.